data/harvard/missing-files.json

```diff
@@ -1,481 +1,568 @@
-        "law.free.cap.coffey.4/357.5775683.json",
-        "law.free.cap.coffey.4/472.5775543.json",
-        "law.free.cap.coffey.4/319.5775443.json",
-        "law.free.cap.coffey.4/10.5775643.json",
-        "law.free.cap.ad3d.54/440.3786514.json",
-        "law.free.cap.ad3d.54/586.3796579.json",
-        "law.free.cap.ny-crim.31/411.1207913.json",
-        "law.free.cap.ny-crim.36/487.1218391.json",
-        "law.free.cap.ny-crim.36/336.1218376.json",
-        "law.free.cap.ny-crim.36/542.1218362.json",
-        "law.free.cap.ny-crim.36/461.1218290.json",
-        "law.free.cap.ny-crim.36/450.1218367.json",
-        "law.free.cap.ad2d.290/728.97369.json",
-        "law.free.cap.ad2d.290/635.97243.json",
-        "law.free.cap.ad2d.290/559.97298.json",
-        "law.free.cap.ad2d.264/716.150711.json",
-        "law.free.cap.ad3d.148/928.12289869.json",
-        "law.free.cap.ad3d.148/985.12290785.json",
-        "law.free.cap.ad3d.148/912.12289545.json",
-        "law.free.cap.ad3d.148/1154.12292995.json",
-        "law.free.cap.ad2d.239/275.1251939.json",
-        "law.free.cap.ad3d.141/1138.6799195.json",
-        "law.free.cap.ad3d.122/497.4308860.json",
-        "law.free.cap.ad3d.125/791.4041760.json",
-        "law.free.cap.ad3d.125/1460.4039090.json",
-        "law.free.cap.ad2d.298/134.271642.json",
-        "law.free.cap.ny-crim.37/74.1220728.json",
-        "law.free.cap.ny-crim.30/214.1206271.json",
-        "law.free.cap.ny-crim.30/209.1206200.json",
-        "law.free.cap.ad3d.140/1068.5771129.json",
-        "law.free.cap.ad3d.140/1012.5774100.json",
-        "law.free.cap.ad3d.140/493.5774767.json",
-        "law.free.cap.ad3d.140/441.5774354.json",
-        "law.free.cap.ad3d.140/617.5773379.json",
-        "law.free.cap.ad3d.140/1100.5770189.json",
-        "law.free.cap.ad3d.147/793.12305790.json",
-        "law.free.cap.ad3d.147/554.12303168.json",
-        "law.free.cap.ad3d.147/589.12303754.json",
-        "law.free.cap.ad3d.147/1326.12309040.json",
-        "law.free.cap.ad3d.147/913.12306873.json",
-        "law.free.cap.ad3d.147/957.12307146.json",
-        "law.free.cap.ad3d.147/750.12305408.json",
-        "law.free.cap.ad3d.147/888.12306683.json",
-        "law.free.cap.cal-3d.6/176.2274674.json"
+        "data/harvard\\law.free.cap.a3d.177\\308.12350253.json",
+        "data/harvard\\law.free.cap.abb-pr.10\\457.1966867.json",
+        "data/harvard\\law.free.cap.abbn-cas.20\\385.1958397.json",
+        "data/harvard\\law.free.cap.ad.242\\737.5834501.json",
+        "data/harvard\\law.free.cap.ad.242\\749.5824702.json",
+        "data/harvard\\law.free.cap.ad.66\\1.4865239.json",
+        "data/harvard\\law.free.cap.ad.66\\12.4865757.json",
+        "data/harvard\\law.free.cap.ad.66\\14.4865567.json",
+        "data/harvard\\law.free.cap.ad.66\\18.4867078.json",
+        "data/harvard\\law.free.cap.ad.66\\21.4864107.json",
+        "data/harvard\\law.free.cap.ad.66\\7.4868211.json",
```

Comment:
something looks off with this file.

Reply:
I think it's because i'm running windows 10, let me run the command with WSL

Reply:
must be - lets include a new command for windows and add it to the docs (assuming that its explained)